src/localization/strings.ts

```diff
@@ -0,0 +1,36 @@
+import en from "./en.json";
+import fi from "./fi.json";
+import LocalizedStrings, { LocalizedStringsMethods } from "localized-strings";
+
+/**
+* Language names interface
+*/
+interface Localizations {
+  en: string;
+  fi: string;
+}
+
+/**
+* Translations interface
+*/
+interface Translations {
+  notYetImplemented: string;
+  currentLocaleLabel: string;
+  localization: Localizations;
+  logout: string;
+}
+
+/**
+* Localized strings
+*/
+export interface Localized extends LocalizedStringsMethods, Translations {}
```

Comment:
I don't think there is any advantage to doing it this way. Typically the interface you have called Translations would be called Localized and extend LocalizedStringsMethods, rather than doing it here as you have done. Let me know if there is a reason for doing it this way I have missed